### PR TITLE
Various small fixes in preparation for the counter PR.

### PR DESCRIPTION
**Pull request by ejj**

### CODE REVIEW

File: quiltctl/ssh/resize/resize.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// +build !windows
```
Comment (kklin) [inline quiltctl/ssh/resize/resize.go]: I thought we aren't targeting windows anymore?

File: minion/docker/docker.go [hunk 2]
```diff
@@ -77,7 +77,6 @@ type RunOptions struct {
 type client interface {
 	StartContainer(id string, hostConfig *dkc.HostConfig) error
 	UploadToContainer(id string, opts dkc.UploadToContainerOptions) error
-	DownloadFromContainer(id string, opts dkc.DownloadFromContainerOptions) error
```
Comment (kklin) [inline minion/docker/docker.go]: Is there an implementation to delete as well?

File: quiltctl/ssh/resize/resize.go (hunk 1 above)
Comment (ejj) [inline quiltctl/ssh/resize/resize.go]: You know, I wrote this patch a long time ago, and the problem it was intended to fix, I don't seem to be having anymore.  I'll likely just drop the patch.

**Comment by kklin** [comment]
LGTM assuming travis passes

**Pull request merged**